### PR TITLE
add DV device option (linux)

### PR DESCRIPTION
**Pull request by privatezero**
Adds option for enabling DV capture device on Linux - will need related documentation updates to reflect installation of dependencies via native package managers.

### CODE REVIEW

File: ffmpegdecklink.rb [hunk 1]
```diff
@@ -105,6 +106,7 @@ def install
     args << "--enable-libfontconfig" if build.with? "fontconfig"
     args << "--enable-libgme" if build.with? "game-music-emu"
     args << "--enable-libgsm" if build.with? "libgsm"
+    args << "--enable-libiec61883" if build.with? "iec61883"
```
Comment (dericed) [inline ffmpegdecklink.rb]: can this `if` also test the OS?

Comment (privatezero) [reply ffmpegdecklink.rb]: If you want it! I was originally going to write it like that, but then I figured that anyone adding that flag would have gotten it out of the Linux install docs, so maybe redundant. I'll add it in though - makes sense I suppose!

Comment (retokromer) [reply ffmpegdecklink.rb]: I presume (untested) that
```rb
args << "--enable-libiec61883" if (build.with? "iec61883") && OS.linux?
```
would work.

Comment (privatezero) [reply ffmpegdecklink.rb]: Does indeed!

**Comment by privatezero** [comment]
Cool! After this is merged, I'll update vrecord docs to reflect.

**Comment by retokromer** [comment]
@dericed Thank you for the feedback!

@privatezero What about putting Linux into the option line?
```rb
option "with-iec61883", "Enable DV device (Linux)" if OS.linux?
```
Sorry, I cannot test it today. If you could… otherwise I will merge during my evening / your morning.

**Pull request merged**